### PR TITLE
Updated READMEs to reflect search algorithms implemented

### PR DESCRIPTION
**Pull request by AlexDvorak**

### CODE REVIEW

**Comment by AnshulMalik** [comment]
Hey, it'd be handy if we can point those to the implementation, so it's easier to find them.

**Comment by AlexDvorak** [comment]
Will do

File: README.md [hunk 1]
```diff
@@ -6,18 +6,20 @@ These are for demonstration purposes only.
...
 ## Sort Algorithms
 
-- Bubble
-- Insertion
+- [Bubble](./sorting/bubble_sort.rs)
+- [Counting](./sorting/counting_sort.rs)
+- [Insertion](./sorting/insertion_sort.rs)
 - Merge _(Not implemented yet)_
-- Quick
+- [Quick](./sorting/quick_sort.rs)
```
Comment (AnshulMalik) [inline README.md]: shouldn't it be `./src/sorting/...`?

Comment (AlexDvorak) [reply README.md]: yes it should be, will change

**Comment by AnshulMalik** [comment]
Thanks @AlexDvorak

**Pull request merged**